src/dash-table/derived/data/virtualized.ts

```diff
@@ -0,0 +1,63 @@
+import * as R from 'ramda';
+
+import { memoizeOneFactory } from 'core/memoizer';
+import {
+    IDerivedData,
+    IUserInterfaceViewport,
+    IUserInterfaceCell,
+    IVirtualizedDerivedData
+} from 'dash-table/components/Table/props';
+
+const getter = (
+    virtualization: boolean,
+    uiCell: IUserInterfaceCell | undefined,
+    uiHeaders: IUserInterfaceCell[] | undefined,
+    uiViewport: IUserInterfaceViewport | undefined,
+    viewport: IDerivedData
+): IVirtualizedDerivedData => {
+    if (!virtualization) {
+        return {
+            ...viewport,
+            offset: { rows: 0, columns: 0 },
+            padding: {
+                rows: { before: 0, after: 0 }
+            }
+        };
+    }
+
+    if (!uiViewport || !uiCell) {
+        return {
+            data: viewport.data.slice(0, 1),
+            indices: viewport.indices.slice(0, 1),
+            offset: { rows: 0, columns: 0 },
+            padding: {
+                rows: { before: 0, after: 0 }
+            }
+        };
+    }
```

Comment:
On first render, only display one row in the table to evaluate the height of the cell and update the state for the real, follow up render